awesome/communities/data/fullremote-it.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "fullremote.it",
+  "url": "https://fullremote.it/",
+  "type": ["Newsletter", "Channel"],
```

Comment:
The "type" key must be a string. 
You can add only one of the two options Newsletter or Channel